.cirrus.yml

```diff
@@ -35,8 +35,14 @@ task:
   build_script:
     - cmake --build Dist -j 2
   test_pcapplusplus_script:
+    - python3 -m pip install gcovr
     - python3 -m pip install -r ci/run_tests/requirements.txt
     - python3 ci/run_tests/run_tests.py --interface eth0
+  coverage_report_script:
+    - gcovr -v -r . --gcov-ignore-parse-errors --exclude-throw-branches --filter Common --filter Pcap --filter Packet --xml -o coverage.xml
```

Comment:
I think there is a GitHub actions for this no ?
https://github.com/codecov/codecov-action

Reply:
You mean for uploading part (L43-L45), right? If yes, it is but I don't know how to use github actions at cirrus. Let me check it.

Reply:
Sorry my comment was for GitHub ci only